dipy/nn/deepn4.py

```diff
@@ -274,7 +274,8 @@ def load_resample(self, subj):
             in_max,
         )
 
-    def predict(self, img, img_affine):
+    @warning_for_keywords()
```

Comment:
not needed since this is new.  So to remove

The decorator is needed for legacy code only

Reply:
can you address this comment @pjsjongsung ?

Reply:
my bad thought I addressed this. Should be gone now